### PR TITLE
add CMP0045 cmake policy before gazebo include in CMakeLists.txt

### PR DESCRIPTION
**Pull request by redkite**
This fixes compile time warnings due to the new cmake policy 0054. Only tested on melodic, since I don't have a working kinetic-setup. Since this was introduced with cmake 3.1 it should work on Ubuntu 16.04 though.

### CODE REVIEW

File: hector_gazebo_plugins/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.1)
```
Comment (meyerj) [inline hector_gazebo_plugins/CMakeLists.txt]: `cmake_minimum_required(VERSION 3.1)` implies `cmake_policy(SET CMP0054 NEW)` if `CMP0054` was introduced in CMake 3.1 (https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html). So one of the two changes is not necessary.

We would like to keep the `kinetic-devel` and newer branches in sync as long as possible. Although both exist they currently point to the same commit. Kinetic specifies CMake 3.0.2 as the minimum version according to [REP 3](http://www.ros.org/reps/rep-0003.html#kinetic-kame-may-2016-may-2021), so depending on 3.1 is strictly speaking not an option if the branches should stay in sync. This patch can easily be written in a way that works with both distributions and older CMake versions:
```cmake
cmake_minimum_required(VERSION 2.8.3)

if(POLICY CMP0054)
  cmake_policy(SET CMP0054 NEW)
endif()
```

Comment (redkite) [reply hector_gazebo_plugins/CMakeLists.txt]: Okay, I'll look into and update this PR.

Comment (redkite) [reply hector_gazebo_plugins/CMakeLists.txt]: @meyerj Have another look, I modified it so that it should also work well with `kinetic-devel`.

**Comment by meyerj** [comment]
Thanks, LGTM.

**Pull request merged**